integration_tests/testsuite/deploy_app.py

```diff
@@ -21,6 +21,8 @@
 import subprocess
 import sys
 
+import test_util
+
 
 def _cleanup(appdir):
```

Comment:
You probably want to add some cleanup code to delete the version. There is a pretty low maximum number of versions for an application.